kover-gradle-plugin/examples/android/dynamic/dyn/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools">
+
```

Comment:
To make this module work as a Dynamic Feature, you must declare the <dist> tag and the install type.

Like this one:
```
<dist:module
    dist:instant="false"
    dist:title="@string/mgm_title">
    <dist:delivery>
        <dist:install-time />
    </dist:delivery>
    <dist:fusing dist:include="false" />
</dist:module>
```